joomla/joomla-cms
# [com_users] groups view: Add enabled/disabled users counting

Pull Request for Improvement.
#### Summary of Changes

This PR, as done in many other views, adds enabled/disabled users counting with links to com_users groups view.

Also:
- Minor code style changes
- Makes the delete validation js more portable (no php vars)
- Adds missing no wraping columns
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14684462/edf9495e-0727-11e6-8bfa-da2713f65fb6.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14684427/ca50c0c2-0727-11e6-9d00-41e5cd92c13b.png)
#### Testing Instructions
1. Use latest staging and apply this patch
2. Go to "Users" -> "Groups"
3. Test the users countings for each group
4. Also create a group and assigna a user to that group, then try to delete that group. First js warning say yes, the second js say no and check the group is not deleted.
